package.json

```diff
@@ -48,7 +48,8 @@
     "lint": "yarn eslint src --ext .ts",
```

Comment:
need to update package.json version!

Reply:
done

Reply:
my bad I committed but forgot to push. Now it's done.